lib/kdeproject_component.rb

```diff
@@ -18,13 +18,15 @@
 # You should have received a copy of the GNU Lesser General Public
 # License along with this library.  If not, see <http://www.gnu.org/licenses/>.
 
-# downloads and makes available as arrays lists of KDE projects which 
+# downloads and makes available as arrays lists of jobs which 
 # are part of Plasma, Applications and Frameworks
 
 require 'httparty'
 
 class KDEProjectsComponent
+  @projects_to_jobs = {'kirigami'=> 'kirigami2', 'discover'=> 'plasma-discover'}
```

Comment:
Space inside { missing.
Surrounding space missing for operator =>.
Space inside } missing.
This should probably be a constant, not a class instance variable.